megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -390,28 +408,29 @@ public void setMiniReportDisplayDialog(final MiniReportDisplayDialog miniReportD
     /**
      * Try to load the "bing" sound clip.
      */
-    private void loadSoundClip() {
-        if (GUIP.getSoundBingFilename() == null) {
-            return;
+    private Clip loadSoundClip(String filename) {
+        Clip clip = null;
```

Comment:
There's no reason to initialize this before the try, which wastes memory if any of the conditions aren't hit.